src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
```

Comment:
Better don't use `@Data` with entities.

https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/

Reply:
ok